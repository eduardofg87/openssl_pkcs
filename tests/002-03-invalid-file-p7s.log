
---- EXPECTED OUTPUT
exception throwed
---- ACTUAL OUTPUT
Fatal error: Invalid File. in /home/php-src/ext/openssl_pkcs/tests/002-03-invalid-file-p7s.php on line 3
---- FAILED
