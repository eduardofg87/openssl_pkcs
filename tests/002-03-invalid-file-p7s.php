<?php
try { 
    $p7s = new Openssl\P7s('tests/001.phpt');
} catch (Exception $e) {
    echo 'exception throwed';
}
