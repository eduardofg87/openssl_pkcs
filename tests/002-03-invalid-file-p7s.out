Fatal error: Invalid File. in /home/php-src/ext/openssl_pkcs/tests/002-03-invalid-file-p7s.php on line 3